charts/kube-prometheus-stack/scripts/update_crds.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
```

Comment:
Mind moving this script to the hack directory?

Reply:
sure, will do